core/new-gui/nx.json

```diff
@@ -18,8 +18,7 @@
           "e2e"
         ],
         "runtimeCacheInputs": [
-          "node -v",
-          "cat ./src/environments/environment.prod.ts"
```

Comment:
we explicitly added this line for a purpose. Please see #1496. If this line is problematic on windows, we should try to fix it.

Reply:
I will convert this PR to a draft until we find a better solution.

Reply:
Ok sounds good.